File: backend/src/main/kotlin/bo/edu/umsa/backend/dto/TaskCommentDto.kt
package bo.edu.umsa.backend.dto

import java.sql.Timestamp

data class TaskCommentDto(
    val taskCommentId: Int,
    val user: UserPartialDto,
    val taskCommentNumber: Int,
    val taskComment: String,
    val txDate: Timestamp,
    val taskCommentFiles: List<FilePartialDto>,
)

File: frontend/src/app/features/task/models/task-comment.dto.ts
import {UserDto} from "../../user/models/user.dto";
import {FileDto} from "../../../core/models/file.dto";

export interface TaskCommentDto {
    taskCommentId: number;
    user: UserDto;
    taskCommentNumber: number;
    taskComment: string;
    txDate: Date;
    taskCommentFiles: FileDto[];
}
